.github/workflows/build.yml

```diff
@@ -4,31 +4,40 @@ on:
     branches:
       - main
       - release/**
+      - jvm-matrix
   pull_request:
     branches:
       - '*'
 
 jobs:
   build:
-    name: Build Job ${{ matrix.os }}
+    name: Build Job ${{ matrix.os }} - ${{ matrix.jvm }}
     runs-on: ${{ matrix.os }}
     strategy:
       # we want that the matrix keeps running, default is to cancel them if it fails.
       fail-fast: false
       matrix:
         # TODO: windows-latest
         os: [ubuntu-latest, macos-latest]
+        # Zulu Community distribution of OpenJDK
+        jvm: [ 8.0.302-zulu, 11.0.12-zulu ]
 
     steps:
       - name: Git checkout
         uses: actions/checkout@v2
 
-      # Zulu Community distribution of OpenJDK
-      - name: set up JDK 1.8
-        uses: actions/setup-java@v2
+      - name: "Download JVM: ${{ matrix.jvm }}"
+        uses: sdkman/sdkman-action@master
+        id: sdkman
         with:
-          distribution: 'adopt'
-          java-version: '8'
+          candidate: java
+          version: ${{ matrix.jvm }}
+
+      - name: "Set up JVM: ${{ matrix.jvm }}"
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+          jdkFile: ${{ steps.sdkman.outputs.file }}
```

Comment:
instead of using the jdkFile, we can also use the java-version, see https://github.com/getsentry/sentry-android-gradle-plugin/blob/main/.github/workflows/test-publish-dry-run.yaml#L66-L70